docs/en/observability/synthetics-analyze.asciidoc

```diff
@@ -0,0 +1,358 @@
+[role="screenshot"]
+image::images/synthetics-analyze-one-run-code-executed.png[Step carousel on a page detailing one run of a browser monitor in the Synthetics UI]
+
+// What info it includes
+Scroll down to dig into the steps in this journey run.
+Click the image:images/icons/arrowRight.svg[Arrow right icon] icon next to the step number to show details.
+The details include metrics for the step in the current run and the step in the last successful run.
+Read more about step-level metrics below in <<synthetics-analyze-one-step-timing>> and
+<<synthetics-analyze-one-step-metrics>>.
+
+++++
+<div style="color:#F04E98">
+++++
+
+// What you might do with this info
+Use this information to...
+
+++++
+</div>
+++++
+
+// Screenshot of the viz
+[role="screenshot"]
+image:images/synthetics-analyze-one-run-compare-steps.png[Step list on a page detailing one run of a browser monitor in the Synthetics UI]
+
+Drill down to see even more details for an individual step by clicking the
+*image:images/icons/apmTrace.svg[Performance breakdown icon]* next to one of the steps. 
+This will take you to the view described below in <<synthetics-analyze-one-step>>.
+
+[discrete]
+[[synthetics-analyze-one-step]]
+== Details for one step
+
+// What is it
+
+// How to get to it
+Click Performance breakdown
+
+[discrete]
+[[synthetics-analyze-one-step-screenshot]]
+=== Screenshot
+
+// What info it includes
+By default the synthetics library will capture a screenshot for each step regardless of
+whether the step completed or failed.
+
+[NOTE]
+====
+Customize screenshot behavior for all monitors in the <<synthetics-configuration>>,
+for one monitor using <<synthetics-monitor-use,`monitor.use`>>, or for a run using
+the <<elastic-synthetics-command, CLI>>.
+====
+
+// What you might do with this info
+Screenshots can be particularly helpful to identify what went wrong when a step fails because of a change to the UI.
+You can compare the failed step to the last time the step successfully completed.
+
+// Screenshot of the viz
+[role="screenshot"]
+image::images/synthetics-analyze-one-step-screenshot.png[Screenshot for one step in a browser monitor in the Synthetics UI]
+
+[discrete]
+[[synthetics-analyze-one-step-timing]]
+=== Timing
+
+The *Timing* visualization shows a breakdown of the time spent in each part of
+the resource loading process for the step including:
+
+* *Blocked*: The request was initiated but is blocked or queued.
+* *DNS*: The DNS lookup to convert the hostname to an IP Address.
+* *Connect*: The time it took the request to connect to the server.
+  Lengthy connections could indicate network issues, connection errors, or an overloaded server.
+* *TLS*: If your page is loading resources securely over TLS, this is the time it took to set up that connection.
+* *Wait*: The time it took for the response generated by the server to be received by the browser.
+  A lengthy Waiting (TTFB) time could indicate server-side issues.
+* *Receive*
```

Comment:
The time it took to receive the response from the server (which can be impacted by the size of the response)